src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -41,17 +41,18 @@ class SentryApplicationRow extends React.PureComponent {
       );
       const query = Object.assign(qs.parse(url.query), installQuery);
       redirectUrl = `${url.protocol}//${url.host}${url.pathname}?${qs.stringify(query)}`;
+      window.location.assign(redirectUrl);
     }
 
-    return redirectUrl;
+    browserHistory.push(redirectUrl);
```

Comment:
I would call this function something else now, if it's actually doing the redirecting.

Reply:
yeah `browserHistory` only takes relative routes afaik and cool can rename